src/js/chartAuthoring.js

```diff
@@ -133,27 +181,43 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             onChartAuthoringInterfaceReady: null,
             onPanelReady: null,
             onPieChartReady: null,
+            onSonifierReady: null,
             onToolReady: {
                 events: {
                     onPanelReady: "onPanelReady",
-                    onPieChartReady: "onPieChartReady"
+                    onPieChartReady: "onPieChartReady",
+                    onSonifierReady: "onSonifierReady"
                 },
                 args: ["{that}"]
             }
         },
         listeners: {
-            "onToolReady.addAriaConnections": "floe.chartAuthoring.addAriaConnections"
+            "onToolReady.addAriaConnections": "floe.chartAuthoring.addAriaConnections",
+            "onToolReady.bindPlayClick": {
+                "this": "{that}.dom.sonifierPlay",
+                "method": "click",
+                "args": ["{chartAuthoringInterface}.sonifier.events.onSonificationRequested.fire"]
+            },
+            "onToolReady.bindStopClick": {
+                "this": "{that}.dom.sonifierStop",
+                "method": "click",
+                "args": ["{chartAuthoringInterface}.sonifier.stopSonification"]
+            }
         },
-        invokers:{
-            "updateDataEntryPanel": {
-                funcName: "floe.chartAuthoring.updateDataEntryPanel",
+        invokers: {
+            "updateDataEntryPanelFromDataSet": {
+                funcName: "floe.chartAuthoring.updateDataEntryPanelFromDataSet",
                 args: ["{that}", "{arguments}.0"]
             },
             "resetDataEntryPanel": {
-                funcName: "floe.chartAuthoring.updateDataEntryPanel",
+                funcName: "floe.chartAuthoring.updateDataEntryPanelFromDataSet",
                 args: ["{that}", []]
             }
         },
+        model: {
```

Comment:
This ```model``` block can be removed as those model paths have been moved to the ```pie``` and ```legend``` components.